src/main/resources/db/changelog/db.changelog-RDM-8463.xsl

```diff
@@ -0,0 +1,17 @@
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
the ticket is requesting to add indexes for all the unindexed JSON search currently on PRod. I can only see we are adding the liquibase scripts for the 4 indexes that were introduced manually on Prod last week. But, are there other indexes that we still need to add? Did we ask Alliu?